mavlink/modules/flight_controller.py

```diff
@@ -101,3 +103,22 @@ def get_home_location(self,
             return False, None
 
         return True, location
+
+    def hover_above_location(self, latitude: float, longitude: float) -> bool:
```

Comment:
I think it would be better if the method signature is something like: `def move_to_position(self, drone_odometry.DronePosition) -> bool:`

The drone should be able to be commanded to any position in 3D space, not tied to a specific altitude.